app/models/manageiq/providers/hawkular/common/event_catcher/stream_mixin.rb

```diff
@@ -17,14 +17,21 @@ def each_batch
 
   def fetch
     new_alerts = []
-    alert_tenants.each do |tenant|
-      log_handle.debug "Fetching tenant [#{tenant}] events using criteria [#{hawkular_alert_criteria}]"
-      new_alerts.concat(@ems.alerts_client(:tenant => tenant).list_alerts(hawkular_alert_criteria))
-    end
+    new_alerts.concat(fetch_criteria(open_alert_criteria))
+    new_alerts.concat(fetch_criteria(resolved_alert_criteria)) if method(:resolved_alert_criteria)
```

Comment:
A few questions:

Is is safe for the mixin to assume open_alert_criteria?

Is there no time criteria on the fetch?  Perhaps this is because of the negative tag query approach?

Instead of two fetches, one for OPEN+ACKNOWLEDGED, and one for RESOLVED, perhaps it would be more efficient to use one fetch, if asking for resolved use no 'statuses' criteria, otherwise statuses=OPEN+ACKNOWLEDGED?